analysis/src/data/get_census_data.py

```diff
@@ -0,0 +1,79 @@
+from optparse import OptionParser
+import pandas as pd
+import geopandas as gpd
+import json
+from pathlib import Path
+import os
+import censusdis.data as ced
+from pandas import json_normalize
+
+
+def get_census(config, city_key):
+    """
+    Retreives census shapefiles for city based on msa codes supplied in config
+    """
+
+    base_path = config['base_path']
+    city_config = json_normalize(config[city_key])
+    msa_codes = city_config[city_config['city_code'] == city_key]['msa_code'].iloc[0]
```

Comment:
Confirming that `iloc[0]` here still returns the msa array and not just the first element of it

Reply:
yes, it does